tests/menus/test_frequently_used_context_menu.py

```diff
@@ -108,7 +104,6 @@ def test_inspect(driver: Firefox):
     title_header = example_page.get_element("title-header")
     example_page.context_click(title_header)
 
-    inspect_option = context_menu.get_context_item("context-menu-inspect")
-    context_menu.click_context_item(inspect_option)
-    context_menu.hide_popup_by_child_node(inspect_option)
+    context_menu.click_and_hide_menu("context-menu-inspect")
+
     devtools.check_opened()
```

Comment:
Wait, how does this verify that the dev tools show? Sorry could you explain how the `DevTools` class is related to the inspector? It seems a bit loose for correctness here

Reply:
`check_opened()` calls `wait_for_page_to_load()`, which executes polling `loaded()` which checks that `requiredForPage` elements exist. Since the devtools panel is required for page for DevTools, just `wait_for_page_to_load()` confirms that DevTools opened